geode-assembly/src/acceptanceTest/java/org/apache/geode/client/sni/ClientSNICQAcceptanceTest.java

```diff
@@ -152,6 +165,21 @@ public void performSimpleCQOverSNIProxy()
 
     assertThat(eventUpdateCounter.get()).isEqualTo(62);
 
+    // verify that the server cleans up when the client connection to the gateway is destroyed
+    ((PoolImpl) cache.getDefaultPool()).killPrimaryEndpoint();
+    // since we can't run code in the server let's grab the CQ statistics and verify that
+    // the CQ has been closed. StatArchiveReader has a main() that we can use to get a printout
+    // of stat values
+    await().untilAsserted(() -> {
+      String stats = docker.exec(options("-T"), "geode",
+          arguments("java", "-cp", "/geode/lib/geode-dependencies.jar",
+              "org.apache.geode.internal.statistics.StatArchiveReader",
+              "stat", "server-dolores/statArchive.gfs", "CqServiceStats.numCqsClosed"));
+      System.out.println("stats from server are :" + stats);
```

Comment:
do we need/want these stats in the test output?

Reply:
Yes, if something goes wrong we can see why things didn't match in the awaitility failure message.  Helped me a lot.